src/applications/vaos/new-appointment/components/VAFacilityPage/EligibilityCheckMessage.jsx

```diff
@@ -3,16 +3,26 @@ import { Link } from 'react-router-dom';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 import ErrorMessage from '../../../components/ErrorMessage';
 import FacilityAddress from '../../../components/FacilityAddress';
+import { ELIGIBILITY_REASONS } from '../../../utils/constants';
 
 export default function EligibilityCheckMessage({
   eligibility,
   facilityDetails,
+  typeOfCare,
   typeOfCareName,
 }) {
-  if (eligibility.requestFailed) {
+  const requestReason = eligibility.requestReasons[0];
```

Comment:
The new data structure for eligibility results has a list of reason codes, we want to display the first one to the user

Reply:
Are these `requestReasons` user friendly codes? I saw that we're logging some readable `reasonMapping` to the console but if these are mapped to `ELIGIBILITY_REASONS ` it seems like they wouldn't be displayed to the user.

Reply:
Ah, scratch that. I'm seeing the updated content based on the `requestReason`.